frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -5,6 +5,7 @@ import {BoardStatusActionService} from "core-app/modules/boards/board/board-acti
 import {BoardVersionActionService} from "core-app/modules/boards/board/board-actions/version/version-action.service";
 import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/query-updated.service";
 import {BoardAssigneeActionService} from "core-app/modules/boards/board/board-actions/assignee/assignee-action.service";
+import {BoardSubprojectActionService} from "core-app/modules/boards/board/board-actions/subproject/subproject-action.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/0ab7334fc73e722331f0dfb4eda5da6f3d906b69/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L9"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>